packages/actions/test/unit/security.test.ts

```diff
@@ -157,6 +162,10 @@ describe("Security", () => {
         })
     })
 
+    afterEach(async () => {
+        await signOut(userAuth)
+    })
+
     describe("GeneratePreSignedURL", () => {
```

Comment:
because you sign out after each test, you need to make sure to login again in the tests that need you to be signed in, otherwise they will fail:

* `should return a pre-signed URL when given the bucket name for a ceremony` -> need you to be signed in 
* all of the tests in multi part upload are broken. Please remove this sign out inside the describe block for multiPartUpload
* some tests in setup are also broken. Looks like all of this signOut is messing up with auth.